node/node.go

```diff
@@ -236,3 +193,19 @@ func (n *Node) EventBus() *lltypes.EventBus {
 func (n *Node) ProxyApp() proxy.AppConns {
 	return n.proxyApp
 }
+
+func newProxyAppValidator(app proxy.AppConns) pubsub.Validator {
```

Comment:
I think `newTxValidator` would be better name, as this is validator for Tx's only.

Reply:
Evan probably chose the name to highlight that this validator talks back to the proxy app. Depends which information is more relevant here (and if the resulting validator is put into a properly named var as well).